src/alfasim_score/constants.py

```diff
@@ -15,3 +19,14 @@
 
 # This default fluid name for packer and fluid above filler
 FLUID_DEFAULT_NAME = "fluid_default"
+
+# nodes data
+BASE_PVT_TABLE_NAME = "base"
+GAS_LIFT_PVT_TABLE_NAME = "gas_lift"
+NULL_VOLUMETRIC_FLOW_RATE = Scalar(0.0, STD_VOLUMETRIC_FLOW_RATE_UNIT)
+NULL_MASS_FLOW_RATE = Scalar(0.0, MASS_FLOW_RATE_UNIT)
+WELL_TOP_NODE_DEFAULT_TEMPERATURE = Scalar(40.0, TEMPERATURE)
```

Comment:
Yeah, it has. Do you think you could just use the default, that's ok to me, I can drop these default values.

Reply:
Those values ares supposed to be defined when defining the well operation, such as production, injection, shut in, etc.
Since it seems temporary, IMHO it makes sense to use the sdk ones.